.github/CODEOWNERS

```diff
@@ -194,10 +194,9 @@ android/java/res @brave/Android
 android/java/org/chromium/chrome/browser/preferences/BravePrefServiceBridge.java @AlexeyBarabash @samartnik @SergeyZhukovsky
 
 # GitHub configuration
-/.github/ @bridiver @cdesouza-chromium
 /.github/CODEOWNERS @bridiver @bsclifton
 /.github/PULL_REQUEST_TEMPLATE.md @bsclifton
-
+/.github/ @brave/devops-team
```

Comment:
this still has to go above CODEOWNERS or it will take precedence over the other two. As the first entry it will match anything except `CODEOWNERS` or `PULL_REQUEST_TEMPLATE.md`